### ISSUE TITLE
Primes Demo is broken

### ISSUE THREAD

**Issue by JamesMilnerUK** [ampproject/worker-dom#503]
At the moment setting the style on `div`s in the prime number demos is throwing an error. 

```
primes.js:1 Uncaught TypeError: this[67] is not a function
    at F.setProperty (primes.js:1)
    at F.set (primes.js:1)
    at generatePrimes (primes.js:67)
    at Object.<anonymous> (primes.js:78)
    at primes.js:81
```

Essentially this (the current code) throws an error:

```javascript
  for (let i = startNumber; i < limit; i++) {
    const numberDiv = document.createElement('div');
    numberDiv.className = 'number';
    const numberText = document.createTextNode(i);
    if (primes.has(i)) {
      // numberDiv.style.fontWeight = 'bold';
      // numberDiv.style.color = '#0379c3';
    }
    numberDiv.appendChild(numberText);
    div.appendChild(numberDiv);
  }
```

Commenting out the two `style` calls fixes this:

```javascript
  for (let i = startNumber; i < limit; i++) {
    const numberDiv = document.createElement('div');
    numberDiv.className = 'number';
    const numberText = document.createTextNode(i);
    if (primes.has(i)) {
      // numberDiv.style.fontWeight = 'bold';
      // numberDiv.style.color = '#0379c3';
    }
    numberDiv.appendChild(numberText);
    div.appendChild(numberDiv);
  }
```
It appears the style API is not working quite as intended?

**Comment by kristoferbaxter** [comment]
Will take a look, and add some tests which appear to be missing.

**Comment by kristoferbaxter** [comment]
Resolved with #505, adding additional tests in a followup since the issue was caused by a mutation of the prototype for the 67th numerical key in CSSStyleDeclaration.
